ynput/OpenPype
# deadline: settings are not blocking extension input

## Changelog Description
Settings are not blocking user input. 

## Additional info
Server settings were ensuring unique name on attribute which was not dictionary having name key and was rather only list with strings.

# Context:
Following issue should be fixed 
![image](https://github.com/ynput/OpenPype/assets/40640033/09517d1b-0576-4253-a2b9-47e2d005340b)


## Testing notes:
1. create server package and upload it to your server
2. make sure you include the new version into your active bundle
